examples/coap-server.c

```diff
@@ -1196,6 +1297,11 @@ main(int argc, char **argv) {
         usage(argv[0], LIBCOAP_PACKAGE_VERSION);
         exit(1);
       }
+    case 'S':
```

Comment:
Above this case-statement now is a `break` missing:

```
coap-server.c: In function ‘main’:
coap-server.c:1296:10: warning: this statement may fall through [-Wimplicit-fallthrough=]
 1296 |       if (!cmdline_read_psk_sni_check(optarg)) {
      |          ^
coap-server.c:1300:5: note: here
 1300 |     case 'S':
      |     ^~~~
```

Reply:
Oops - missed that during the rebase - updated.  Changes pushed.